lib/stdlib/src/features.erl

```diff
@@ -0,0 +1,308 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2021-2022. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+-module(features).
+
+-export([features/0,
+         feature_info/1,
+         enabled_features/0,
+         is_valid_feature/1,
+         reserved_words/0,
+         reserved_words/1,
+         resword_add_feature/2,
```

Comment:
is resword a short name for reserved_word? it was not obvious for me at first.

- if yes, then maybe could be used in all function names (e.g. `reswords` rather than `reserved_words`) ?
- I'm wondering if there is a better short name than resword ... I thought about keyword, which might be more obvious, but I'm not sure if they're equivalent in this context. are they?

Reply:
Yes, _keyword_ is both shorter and less cumbersome.  In the code both terms are used for the same concept, so using _keyword_ shouldn't be too controversial.  I'll do some archeology on terms.